colossalai/inference/infer_struct.py

```diff
@@ -0,0 +1,172 @@
+import enum
+from dataclasses import dataclass
+from typing import Dict, List, Set
+
+
+class RequsetStatus(enum.Enum):
+    """The status of Sentences"""
+
+    WAITING = enum.auto()
+    RUNNING = enum.auto()
+    ABORTED = enum.auto()
+    OVERLENGTH = enum.auto()
+    COMPLETED = enum.auto()
+    LENGTH_CAPPED = enum.auto()
+
+    @staticmethod
+    def is_finished(status: "RequsetStatus") -> bool:
+        return status in [
+            RequsetStatus.OVERLENGTH,
+            RequsetStatus.COMPLETED,
+            RequsetStatus.LENGTH_CAPPED,
+        ]
+
+    @staticmethod
+    def is_running(status: "RequsetStatus") -> bool:
+        return status == RequsetStatus.RUNNING
+
+    @staticmethod
+    def is_waiting(status: "RequsetStatus") -> bool:
+        return status == RequsetStatus.WAITING
+
+
+class Sequence:
+    """Store information of input sequence.
+
+    Args:
+        request_id: The ID of input sequence.
+        prompt: The prompt of input sequence.
+        token_id: The ID of input sequence.
+        block_size: The block size of input sequence.
+        sample_params: The sample_params of input sequence.
+        block_table_index: The index of input sequence in block_table.
+        
+        
+        TODO To be modified as needed.
```

Comment:
Same here, people may wondering what `as needed` actually refers to. `modify as needed` is too vague and any developer reading it will have no idea what to do even though it is supposed to be a `TODO` task. An example of clear `TODO` task will be like `delete xxx when xxx is refactored`. 

Also, do not put `TODO` in docstring as docstring is part of the documentation that users will read as well.

Reply:
Done